searchcore/src/apps/vespa-feed-bm/vespa_feed_bm.cpp

```diff
@@ -1313,6 +1318,7 @@ App::get_options()
     enum longopts_enum {
         LONGOPT_CLIENT_THREADS,
         LONGOPT_INDEXING_SEQUENCER,
+        LONGOPT_MAX_PENDING,
```

Comment:
Usage message is not updated, and order is inconsistent with option array.

Reply:
Fixed

Reply:
"documents" is before  "max-pending" in long_opts array, thus LONGOPT_MAX_PENDING should be after LONGOPT_DOCUMENTS in longopts_enum.

Reply:
Now they should all be in alphabetical ordering.